src/utils/inMemoryDB.ts

```diff
@@ -9,18 +9,19 @@ import { logger } from '../logger.js'
  * More portable than requiring user to set up Mongo in a background Docker process.
  * Need a replset to faciliate transactions.
  */
-const mongod = await MongoMemoryReplSet.create({
-  // Stream listener listens on DB denoted by 'MONGO_DBNAME' env var.
-  replSet: { count: 1, storageEngine: 'wiredTiger', dbName: checkVar('MONGO_DBNAME') }
-})
+let mongod: MongoMemoryReplSet
 let stream: ChangeStream
 
 /**
  * Connect to the in-memory database.
  */
 const connect = async (): Promise<void> => {
+  mongod = await MongoMemoryReplSet.create({
+    // Stream listener listens on DB denoted by 'MONGO_DBNAME' env var.
+    replSet: { count: 1, storageEngine: 'wiredTiger', dbName: checkVar('MONGO_DBNAME') }
+  })
   const uri = await mongod.getUri(checkVar('MONGO_DBNAME'))
-  logger.info(`Connecting to database ${uri}`)
+  logger.info(`Connecting to in-memory database ${uri}`)
```

Comment:
To disambiguate from the other tests which rely on the Docker-initiated, disk-based Mongo instance.